client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -167,3 +131,56 @@
 .sticky-inner-wrapper {
   z-index: 1;
 }
+
+.question_type {
+  &--select {
+    border-color: color('white', 10);
+    background-color: color('teal', 40);
+    color: color('white', 10);
+    background-image: url(triangle(color('white', 10), $size: 1.5));
```

Comment:
Not sure what your preference is, but you could do `background: url(blah) color(blah)`

Reply:
i just hope this all goes away once design takes a look at it - for now im going to leave this because it prevents me from having to position the background image (just changes the color of the image)